cli/tests/test_no_abandoned_test_files.rs

```diff
@@ -0,0 +1,24 @@
+use std::fs;
+use std::path::PathBuf;
+
+#[test]
+fn test_no_abandoned_test_files() {
```

Comment:
Optional: If that's easy, I'd consider putting this directly into `runner.rs`.

**Update:** Yuya's suggestion in https://github.com/martinvonz/jj/pull/2914/commits/5daccacfd890cae1c6161ce0a76c9da0c90edca9#r1476082844 sounds good to me as well.

Reply:
If it's not easy, I'd put a comment next to this test in `runner.rs`. If you don't, I can do it myself after this is merged.